res/country_metadata/popularReligions.yml

```diff
@@ -0,0 +1,70 @@
+# religions that should be shown first before the order defined in the form.
+# no total order required. Just mention the religions that should show within the first few entries
+default: []
+AE: [muslim]
+AF: [muslim]
+AZ: [muslim]
+BD: [muslim, hindu]
+BF: [muslim]
+BH: [muslim]
+BN: [muslim]
+BT: [buddhist]
+CA: [christian, muslim, jewish]
+CN: [buddhist, christian, taoist, muslim]
+CZ: [christian, jewish]
+DJ: [muslim]
+DZ: [muslim]
+EG: [muslim]
+GB: [christian, muslim, jewish, sikh]
+GM: [muslim]
+GN: [muslim]
+ID: [muslim, christian, hindu, buddhist]
+IL: [jewish, christian, muslim]
+IN: [hindu, christian, muslim, buddhist, sikh, jain]
+IQ: [muslim]
+IR: [muslim]
```

Comment:
You know where Bahai Faith is from?

Reply:
According to Wikipedia

> It initially grew in Iran (Persia) and other parts of the Middle East region, where it has faced ongoing persecution since its inception.[2] Currently it has between 5 and 7 million adherents, known as Bahá'ís, spread out into most of the world's countries and territories

Given persecution in their home area, spread across world and low total numbers it was not surprising that they are not appearing on list of any country (though it may be next weakness of OSM data).

Reply:
https://en.wikipedia.org/wiki/Iran#Religion

> according to statistics center of Iran, Bahais constitute only about 0.37% of Iran

last data mentioned in https://en.wikipedia.org/wiki/Bah%C3%A1%27%C3%AD_Faith_in_Iran is from 1920

Reply:
The first sentence of this article says "The Bahá'í Faith in Iran is the country's second-largest religion after Islam"

Reply:
somehow I missed it, I added now bahai in Iran